go/go.mod

```
@@ -13,11 +13,10 @@ require (
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/common v0.10.0
 	github.com/prometheus/prometheus v2.5.0+incompatible
-	github.com/sirupsen/logrus v1.6.0
-	github.com/stretchr/testify v1.6.1
-	github.com/testcontainers/testcontainers-go v0.9.0
+	github.com/sirupsen/logrus v1.7.0
```

In case you missed it, but this is the latest available. 
```suggestion
	github.com/sirupsen/logrus v1.8.1
```

---

I just manually updated `testcontainers-go`, the rest is the result of `go mod tidy`. (disclaimer, I am not SUPER familiar with go, so this might have been a mistake)